devtools/maven/src/main/java/io/quarkus/maven/CreateProjectMojo.java

```diff
@@ -294,6 +298,8 @@ public void execute() throws MojoExecutionException {
                     .extensions(extensions)
                     .example(example)
                     .noCode(noCode)
+                    // TODO: Something like this ? Only concern would be if sending the file or just the relative path to be loaded in the command
+                    .parentProject(parentPomModel.getPomFile())
```

Comment:
Maybe the `Path` instead of the `File`

Reply:
the parent relative path as String would also be fine I guess and maybe better as it is then used in the pom.xml, right?